website/pages/ar/deploying/deploy-using-subgraph-studio.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Deploy Using Subgraph Studio
+---
+
+Learn how to deploy your subgraph to Subgraph Studio.
+
+> Note: When you deploy a subgraph, you push it to Subgraph Studio, where you'll be able to test it. It's important to remember that deploying is not the same as publishing. When you publish a subgraph, you're publishing it on-chain.
+
+## Subgraph Studio Overview
+
+In [Subgraph Studio](https://thegraph.com/studio/), you can do the following:
+
+- View a list of subgraphs you've created
+- Manage, view details, and visualize the status of a specific subgraph
+- Create and manage your API keys for specific subgraphs
+- Restrict your API keys to specific domains and allow only certain Indexers to query with them
+- Create your subgraph through the Studio UI
```

Comment:
```suggestion
- Create a new subgraph
```
(I think "through the Studio UI" is redundant because the list is introduced with "In Subgraph Studio, you can do the following:")